terminus-ui/tabs/src/collection/tab-collection.component.spec.ts

```diff
@@ -587,7 +587,6 @@ describe(`TsTabCollectionComponent`, function() {
       secondLabel.click();
       fixture.detectChanges();
       tick();
-      fixture.detectChanges();
 
```

Comment:
why did you take detectChanges away after tick? couple of times i thought detectChanges was necessary after tick.

Reply:
It didn't seem to be needed post-tick. There are some cases it could be needed for though so this isn't a broad rule. I believe it's only needed if the next tick may change something internal during the tick